Dockerfile

```diff
@@ -2,37 +2,34 @@
 FROM python:3.9-slim-buster as builder
 
 # Set environment variables
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1
 
 # Set work directory
 WORKDIR /server
 
-# Install system dependencies
-RUN apt-get update && apt-get install -y build-essential
-
-# Install Python dependencies
+# Install system dependencies and Python dependencies
 COPY ./server/requirements.txt /server/
-RUN pip wheel --no-cache-dir --no-deps --wheel-dir /server/wheels -r requirements.txt
+RUN apt-get update && apt-get install -y build-essential --no-install-recommends \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
Same as above

Reply:
Carryover form original file, will remove and test.